Kelvin_Lau_Project1.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Kelvin_Lau_Project1</title>
+    <link href="../CSS/style.css" rel="stylesheet">
```

Comment:
Add the alt attribute on img tags (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img)